pkg/parser/ast/dml.go

```diff
@@ -3100,6 +3101,7 @@ type ShowStmt struct {
 	ShowProfileLimit *Limit // Used for `SHOW PROFILE` syntax
 
 	ImportJobID *int64 // Used for `SHOW IMPORT JOB <ID>` syntax
+	SQL         string
```

Comment:
1. Please add comments as above.
2. Since it's a string type and the content could be a SQL text or SQL digest, I think it's better to rename it to other names like `SQLOrDigest` or `Target` or add some comments.

Reply:
Good suggestion, added.